recipes/onedpl/all/conanfile.py

```diff
@@ -14,9 +14,12 @@ class OneDplConan(ConanFile):
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/oneapi-src/oneDPL"
     topics = ("stl", "parallelism")
-    settings = "os", "arch", "build_type", "compiler"
```

Comment:
you have to keep this. `self.info.header_only()` is properly called below, so multiple packages are not created anyway